slackeventsapi/__init__.py

```diff
@@ -7,7 +7,11 @@ class SlackEventAdapter(BaseEventEmitter):
     # If no endpoint is provided, default to listening on '/slack/events'
     def __init__(self, signing_secret, endpoint="/slack/events", server=None, **kwargs):
         BaseEventEmitter.__init__(self)
-        self.signing_secret = signing_secret
+        if signing_secret:
+            self.signing_secret = signing_secret
+        else:
+            raise ValueError(f'Got {type(signing_secret)} instead of the Slack signing secret token. Check the first '
+                             f'parameter.')
```

Comment:
How about doing like this?
```suggestion
        if signing_secret is None:
            message = "signing_secret is required but you passed None in the first argument"
            raise ValueError(message)
        self.signing_secret = signing_secret
```

Reply:
Yes, that looks more Pythonic.